hosts/default.nix

```diff
@@ -11,7 +11,7 @@
     # shared modules
     azure-common = import ./azure-common.nix;
     qemu-common = import ./qemu-common.nix;
-    ficolo-hosts = import ./ficolo-hosts.nix;
+    ficolo-common = import ./ficolo-common.nix;
```

Comment:
Not a new issue with this PR, but it seems wrong to me that ficolo-specific (even if common to all ficolo servers) configuration is in hosts/default.nix common to all servers.

Reply:
Thinking more about it, what used to be "wrong" turns to "breaking things" with this PR. We don't want to set those dns servers for machines outside ficolo.

Reply:
Afaik this just "declares" the 'ficolo-common' as importing ficolo-common.nix.
As long as e.g. azure machines do not use 'ficolo-common' at all, they won't include any ficolo stuff into configuration.
And if this breaks things, it was broken before this PR.

Reply:
Yes. This is how @tervis-unikie put it. This just 'introduces' or 'declares' a new module. You still need to include it separately to take it into use with any specific configuration.